cpp/tests/io/parquet_test.cpp

```diff
@@ -881,9 +881,7 @@ TEST_F(ParquetChunkedWriterTest, SingleTable)
   auto filepath = temp_env->get_temp_filepath("ChunkedSingle.parquet");
   cudf_io::chunked_parquet_writer_options args =
     cudf_io::chunked_parquet_writer_options::builder(cudf_io::sink_info{filepath});
-  auto state = cudf_io::write_parquet_chunked_begin(args);
-  cudf_io::write_parquet_chunked(*table1, state);
-  cudf_io::write_parquet_chunked_end(state);
+  cudf_io::parquet_chunked_writer(args).write(*table1).close();
```

Comment:
What do you think about not calling `close` unless needed? It's not needed whenever the writer is a temporary, and I don't think calling it and discarding the data adds coverage.